src/LodeRunner.API/src/Handlers/ResultHandler.cs

```diff
@@ -26,18 +26,10 @@ public static class ResultHandler
         /// <summary>
         /// Creates No Content Result.
         /// </summary>
-        /// <param name="statusCode">Http StatusCode.</param>
         /// <returns>JsonResult.</returns>
-        public static async Task<JsonResult> CreateNoContent(HttpStatusCode statusCode = HttpStatusCode.NoContent)
+        public static async Task<NoContentResult> CreateNoContent()
         {
-            return await Task.Run(() =>
-            {
-                return new JsonResult(null)
-                {
-                    StatusCode = (int)statusCode,
-                    ContentType = JsonContentTypeApplicationJson,
-                };
-            });
+            return await Task.Run(() => new NoContentResult());
```

Comment:
Since this is a single line method, we should just replace this function call where used with the single line:
```csharp
 return await Task.Run(() => new NoContentResult());
```

Look for other similar single line methods of a similar nature that we could refactor out in favor of the direct call.

Reply:
This will be addressed in a separate issue https://github.com/orgs/retaildevcrews/projects/12#card-76493062